src/components/icon/icon.jsx

```diff
@@ -6,14 +6,18 @@ import styles from './icon.styles.pcss';
 import IMAGES from './icons';
```

Comment:
i think it will import all icons 🤔 
make IconButton Component like in Visxa pls